Remove dupe dependencies; upgrade cookie-policy and global-nav

barrymcgee opened the pull request:
## Done

I noticed renovate flagging duplicate dependencies between "dependencies" and "devDependencies" - so I deduped.

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [http://0.0.0.0:8002/](http://0.0.0.0:8002/)
4. Test command `./run build`
5. Test command `./run clean`
6. 4. Test command `./run test`
7. Verify no errors.

webteam-app commented:
Starting demo at: https://www-canonical-com-canonical-websites-pr-334.run.demo.haus/

barrymcgee commented:
@pmahnke I've now fixed up all merge conflicts, split out the deps and devDeps and made sure all deps are pinned to their latest version.

I also then had to make changes to make sure updated cookie-policy and global-nav work.

Also updated 500.html nav as it was broken.

This all merits a full QA again please.

nottrobin commented on static/sass/styles.scss:
```diff
@@ -4,10 +4,9 @@ $color-brand: #e95420;
 @import 'modules/cookie-policy';
 
-// import vanilla-brochure-theme
+// import vanilla
```
So much attention to detail :open_mouth:

barrymcgee replied:
Look after the pennies and the pounds look after themselves. ;)
